content/gherkin/reference.md

```diff
@@ -409,3 +409,6 @@ In order to allow Gherkin to be written in a number of languages, the English ke
 A list of the currently supported languages and their keywords can be found below.
 
 {{% gherkin-i18n-table %}}
+
+## Gherkin Rule
+The **Rule** keyword is optional and it is simply a grouping construct. It can be used to group Scenarios that are related to the same Rule.
```

Comment:
Several requests:
* Please mark `Rule` with back ticks (`), like the other keywords, for consistent formatting. In that case it can be upercase, for consistency.
* Please use lowercase in a sentence (in this case "Scenario" and "Rule") for consistency.
* Please add this info in the correct spot. Imho it should be under "Keywords". The keyword should be listed in the bullit list of keywords in the proper place (probably between Feature and Example). It should then be explained in it's own section in that same order.
* Heading for that section can be "Rule" (for consistency).
* Suggested text edit: "The `Rule` keyword is an optional keyword. It can be used to group scenarios that exemplify the same rule." or something similar. I'm not sure whether there is any other relevant information to mention in this section (from the linked issues).